index.html

```diff
@@ -63,23 +63,221 @@ <h1 class="primary">
+                            <li class="tag">JS</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">Ruby & Rails</li>
+                        </ul>
+                        <button class="butn" type="button" title="see-project">
+                            See Project
+                          </button>
+                    </div>
+                </li>
+                <li class="card">
+                    <img src="assets/images/Img-plaholder.svg" alt="Img-plaholder"/>
+                    <div class="card-text">
+                        <h3 class="project-title">
+                            Multi-Post Stories <br />
+                            Gain+Glory
+                        </h3>
+                        <ul class="tags">
+                            <li class="tag">HTML</li>
+                            <li class="tag">JS</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">Ruby & Rails</li>
+                        </ul>
+                        <button class="butn" type="button" title="see-project">
+                            See Project
+                          </button>
+                    </div>
+                </li>
+                <li class="card">
+                    <img src="assets/images/Img-plaholder.svg" alt="Img-plaholder"/>
+                    <div class="card-text">
+                        <h3 class="project-title">
+                            Multi-Post Stories <br />
+                            Gain+Glory
+                        </h3>
+                        <ul class="tags">
+                            <li class="tag">HTML</li>
+                            <li class="tag">JS</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">Ruby & Rails</li>
+                        </ul>
+                        <button class="butn" type="button" title="see-project">
+                            See Project
+                          </button>
+                    </div>
+                </li>
+                <li class="card">
+                    <img src="assets/images/Img-plaholder.svg" alt="Img-plaholder"/>
+                    <div class="card-text">
+                        <h3 class="project-title">
+                            Multi-Post Stories <br />
+                            Gain+Glory
+                        </h3>
+                        <ul class="tags">
+                            <li class="tag">HTML</li>
+                            <li class="tag">JS</li>
+                            <li class="tag">CSS</li>
+                            <li class="tag">Ruby & Rails</li>
+                        </ul>
+                        <button class="butn" type="button" title="see-project">
+                            See Project
+                          </button>
+                    </div>
+                </li>
+            </ul>
```

Comment:
- [ Required ] I think it would be better if you made sure that the borders of the project cards are the same `size` and `color` as those in Figma so that your design matches the Figma design. The borders in your design differ significantly in color and size from those in Figma please see the screenshot below for more details on this.

- [ Required ] Still on the project cards the `margin-bottom` on the `See project` button is smaller in your design compared to the `margin-bottom` on the Figma design. Please consider rectifying this for all your project cards. 

| Figma design | Your design |
|-------------------|------------------|
|![Screenshot from 2023-06-03 03-07-47](https://github.com/yesuf023/Portfolio/assets/16392046/b5a075da-c72f-48b5-a9e8-684b0172a446)|![Screenshot from 2023-06-03 03-06-07](https://github.com/yesuf023/Portfolio/assets/16392046/574445cb-6cdb-4011-a199-56ce434038af)|